services/core-web/src/styles/generic/layout.scss

```diff
@@ -291,7 +291,7 @@ body {
   }
 }
 
-.padding-large {
+.padding-lg {
```

Comment:
Can we please change this to "padding-hg"?

Reply:
we can change xxl to hg